### PR TITLE
Add Manual Party Info and tests

### PR DESCRIPTION
**Pull request by BobBorges**
- **feat: data from new query**
  

- **feat: add manually checked quality controll data**
  

- **feat: add test to check independent MPs**
  

- **feat: run new test**
  

- **chore: requery wikidata, run existing tests (ok)**
  

- **fix:rm errors**
  

- **chore:requery after update**
  

- **fix: wrong peter**
  

- **feat: add list of people who don't belong in metadata, case 0 - peter lamburg the artist**
  

- **chore:requery**
  

- **chore:merge data from query branch**
  

- **doc: change job name**
  

- **chore:fetch from query branch**
  

- **chore:fetch from query branch**
  

- **chore:fetch from query branch**
  

- **feat: add test file**
  

- **feat: test known party affiliation**
  

- **chore: requery after wd updates**
  

- **fix: , --> ;**
  

- **style: rm debug prints**
  

- **chore: requery wd after edits**
  

- **chore: fetch data from requery branch**
  

- **style: rename job**
  

- **chore: unskip older test**

### CODE REVIEW

**Comment by BobBorges** [comment]
We don't have specific PR tests yet, but all tests ran on push (and passed).

**Comment by MansMeg** [comment]
Great! I guess then they can be merged?

**Comment by BobBorges** [comment]
merge and release 1.1.0

**Comment by MansMeg** [comment]
This is merging to dev. Right?

**Comment by BobBorges** [comment]
it is, yeah. We don't have a release workflow yet, so there will be at least one more pr to dev before we merge to main + release.

**Review by ninpnin — APPROVED** [review]
LGTM

File: data/explicit_no_party.csv
[the remark this answers is not in the document]
Comment (BobBorges) [reply data/explicit_no_party.csv:1]: no -- it's data pulled from wikidata, not a test file

[the remark this answers is not in the document]
Comment (MansMeg) [reply data/explicit_no_party.csv:1]: Ok. I couldnt find it in the readme.

[the remark this answers is not in the document]
Comment (BobBorges) [reply data/explicit_no_party.csv:1]: of course 
image

**Pull request merged**